cs-engine/1.13/index.md

```diff
@@ -50,109 +63,157 @@ to update its RHEL kernel.
 
-1. Log into the system as a user with root or sudo permissions.
-
-2.  Add Docker's public key for CS packages:
+1.  Install packages to allow `apt` to use a repository over HTTPS:
 
     ```bash
-    $ curl -s 'https://sks-keyservers.net/pks/lookup?op=get&search=0xee6d536cf7dc86e2d7d56f59a178ac6c6238f52e' | sudo apt-key add --import
+    $ sudo apt-get update
+
+    $ sudo apt-get install --no-install-recommends \
+        apt-transport-https \
+        curl \
+        software-properties-common
     ```
 
-3.  Install the HTTPS helper for apt (your system may already have it):
+    Optionally, install additional kernel modules to add AUFS support.
 
     ```bash
-    $ sudo apt-get update && sudo apt-get install apt-transport-https
+    $ sudo apt-get install -y --no-install-recommends \
+        linux-image-extra-$(uname -r) \
+        linux-image-extra-virtual
     ```
 
-4.  Install additional kernel modules to add AUFS support.
+2.  Download and import Docker's public key for CS packages:
 
     ```bash
-    $ sudo apt-get install -y linux-image-extra-$(uname -r) linux-image-extra-virtual
+    $ curl -fsSL 'https://sks-keyservers.net/pks/lookup?op=get&search=0xee6d536cf7dc86e2d7d56f59a178ac6c6238f52e' | sudo apt-key add -
     ```
 
-5.  Add the repository for the new version:
+3.  Add the repository. In the  command below, the `lsb_release -cs` sub-command
+    returns the name of your Ubuntu version, like `xenial` or `trusty`.
 
-    for 14.04:
-
-    ```bash
-    $ echo "deb https://packages.docker.com/1.13/apt/repo ubuntu-trusty main" | sudo tee /etc/apt/sources.list.d/docker.list
-    ```
-    
-    for 16.04:
-    
     ```bash
-    $ echo "deb https://packages.docker.com/1.13/apt/repo ubuntu-xenial main" | sudo tee /etc/apt/sources.list.d/docker.list
+    $ sudo add-apt-repository \
+       "deb https://packages.docker.com/1.13/apt/repo/ \
+       ubuntu-$(lsb_release -cs) \
+       main"
     ```
 
-6.  Run the following to install commercially supported Docker Engine and its
-dependencies:
+4.  Install CS Docker Engine:
 
-    ```bash
-    $ sudo apt-get update && sudo apt-get install docker-engine
-    ```
+    - **Latest version**:
+
+      ```bash
+      $ sudo apt-get update
+
+      $ sudo apt-get -y install docker-engine
+      ```
+
+    - **Specific version**:
+
+      On production systems, you should install a specific version rather than
+      relying on the latest.
+
+      1.  List the available versions:
+
+          ```bash
+          $ sudo apt-get update
+
+          $ apt-cache madison docker-engine
+          ```
+
+          The second column represents the version.
 
-7. Confirm the Docker daemon is running:
+      2.  Install a specific version by adding the version after `docker-engine`,
+          separeated by an equals sign (`=`):
+
+          ```bash
+          $ sudo apt-get install docker-engine=<version>
+          ```
+
+5. Confirm the Docker daemon is running:
```

Comment:
The snippet below is not rendering properly. I think you're missing a space here.